salt/runners/event.py

```diff
@@ -76,5 +76,6 @@ def send(tag, data=None):
     '''
     data = data or {}
     event = salt.utils.event.get_master_event(__opts__, __opts__['sock_dir'],
-                                              listen=False)
+                                              listen=False,
+                                              keep_loop=True)
```

Comment:
Can you explain this change a little more please?

Reply:
when keep_loop is false the event destroy() also calls io_loop.close(), I didn't see the need for that to happen in this case. that said I don't think leaving this out would cause any issues either.